CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- The Network Server now provides the timestamp at which it received `JoinAccept` and `ApplicationUplink` messages.
```

Comment:
```suggestion
- The Network Server now provides the timestamp at which it received join-accept or data uplink messages
```

or

```suggestion
- The Network Server now provides the timestamp at which it received `JoinAccept` and `UplinkMessage` messages.
```

In any case:
a. NS does not receive `ApplicationUplink`
b. I don't think we should use proto message names here - we are writing this for users, they should need to look through our source code to understand what a particular change means